Tests/IntegrationTestKernel.php

```diff
@@ -25,7 +25,7 @@ class IntegrationTestKernel extends Kernel
 
     public function __construct(string $environment, bool $debug)
     {
-        $this->randomKey = rand(100, 999);
```

Comment:
This does fix the local runs for me. I got zero errors with Php 8.1 and 8.2 local.

Reply:
This view in Github is misleading. I changed it to `$this->randomKey = rand(10000000000, 99999999999);`
Not a real solution, but good enough for the moment i guess.

Reply:
How is it misleading? I understood that was what you did. Nice find, it would make sense that this is responsible for random failures.

Reply:
Github only shows in the preview that the line was removede, not that it is replaced.